tests/unit_tests/test_app_sentiment.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from app.sentiment import sentiment_score, sentiment_rank, apply_sentiment
+
+
+class TestSentiment(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """Initialize environment for test suite"""
+        cls.text_positive = "Wow! This is super!"
+        cls.text_negative = "Very disappointed. Bad quality!"
+        cls.text_neutral = "No opinion"
+        cls.text_null = ""
+        cls.text_num = "-12345.50"
+
+    def test_sentiment_score_input(self):
+        self.assertIsInstance(sentiment_score(self.text_null), float)
+        self.assertIsInstance(sentiment_score(self.text_num), float)
+        self.assertIsInstance(sentiment_score(self.text_neutral), float)
+
+    def test_sentiment_rank_sentiments(self):
+        self.assertEqual(sentiment_rank(self.text_positive), "Positive")
+        self.assertEqual(sentiment_rank(self.text_negative), "Negative")
+        self.assertEqual(sentiment_rank(self.text_neutral), "Neutral")
+
+    def test_apply_sentiment_update(self):
+        with self.assertRaises(KeyError): apply_sentiment({})
```

Comment:
There's a small PEP8 violation here. After the `:` insert a new line.

Reply:
Fixed.